test/proposals/ERC1046/TokenMetadata.test.js

```diff
@@ -1,33 +1,18 @@
-import assertRevert from '../../helpers/assertRevert';
 const ERC20WithMetadata = artifacts.require('ERC20WithMetadataMock');
 
 require('chai')
   .use(require('chai-as-promised'))
   .should();
 
 const metadataURI = 'https://example.com';
-const invalidMetadataURI = '';
 
 describe('ERC20WithMetadata', function () {
-  context('invalid metadata', function () {
-    before(async function () {
-      this.token = await ERC20WithMetadata.new(invalidMetadataURI);
-    });
-
-    it('throws', async function () {
-      await assertRevert(
-        this.token.tokenURI()
-      );
-    });
+  before(async function () {
```

Comment:
See https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1021. I'm fine with merging and fixing afterwards though because there is only one tests here.